LICENSE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 MercadoLibre.com
```

Comment:
@battaglr Good catch. Fixed.
BTW, you can add any fixes directly to the `license` branch.